src/app/scripts/cac/control/cac-control-directions-form.js

```diff
@@ -95,6 +95,12 @@ CAC.Control.DirectionsFormControl = (function ($, Typeahead, Geocoder, UserPrefe
         events.trigger(eventNames.selected, [key, location]);
     }
 
+    // For setting origin or destination from code, e.g. directions links
+    function setLocation(key, location, text) {
```

Comment:
Whoops.  I changed things around a little and must have checked it with a stale tab.  Filling in the typeahead was definitely the intention.  Stand by...